common/shlibs

```diff
@@ -3987,7 +3987,7 @@ libocl.so.2019.07 opencamlib-2019.07_1
 libcglm.so.0 cglm-0.7.6_1
 libfcft.so.3 fcft-2.2.2_1
 libaml.so.0 aml-0.1.0_1
-libneatvnc.so.0 neatvnc-0.2.0_1
+libneatvnc.so.0 neatvnc-0.3.2_1
```

Comment:
This change isn't necessary, and, if it were, shouldn't be in this PR / in this commit. Otherwise, good to merge.

Reply:
This was probably a git accident, should be easily solvable.